docs/concepts/configuration/assign-pod-node.md

```diff
@@ -206,6 +206,31 @@ If defined but empty, it means "all namespaces."
 All `matchExpressions` associated with `requiredDuringSchedulingIgnoredDuringExecution` affinity and anti-affinity
 must be satisfied for the pod to schedule onto a node.
 
+#### More Practical Usage
+
+Than directly using with pods Interpod Affinity and AnitAffinity will be more useful when they are used with higher
```

Comment:
I would write this as:
Interpod Affinity and AntiAffinity can be even more useful when used with higher level collections such as ReplicaSets, StatefulSets, Deployments, etc.